controllers/admin/AdminOutstandingController.php

```diff
@@ -43,7 +43,8 @@ public function __construct()
 		CONCAT(LEFT(c.`firstname`, 1), \'. \', c.`lastname`) AS `customer`,
 		c.`outstanding_allow_amount`,
 		r.`color`,
-		rl.`name` AS `risk`';
+		rl.`name` AS `risk`,
+		`company`';
```

Comment:
It would be great to write the table (or its alias) before the column. If somebody needs to have another look to this SQL request, he won't know in which table the company is stored.